### DISCUSSION TITLE
Help with plotting weighted Burgers vectors orientations as contoured IPF plot in crystal coordinates

### DISCUSSION THREAD

**Discussion by dsfrei** [mtex-toolbox/mtex#1950] [Ask Anything]
Hello MTEX Community,

I have been calculating WBV using weightedBurgerVec function from ebsd datasets of deformed wrought aluminum. I would like to visual the orientations of the WBVs as a contours in an inverse pole figure (IPF) plot. Something like the below figure produced in Aztec software (https://www.ebsd.com/ois-ebsd-system/dislocation-density-analysis).

My attempt:
```matlab
% W matrix of WBV vectors in crystal coordinates 
W = inv(ebsd.orientations) .* weightedBurgersVec(ebsd)

%plot ipf
plot(W.symmetrise, 'fundamentalRegion') 
```
This does not produce intended results. Could someone please suggest a possible solution? 

Thank you for you time and best regards,
Dan 

Example from AZtec  (https://www.ebsd.com/ois-ebsd-system/dislocation-density-analysis). The goal would be to create something like this, but for (FCC aluminum)

![image](https://github.com/mtex-toolbox/mtex/assets/139288012/2656ea34-761a-49a9-8dcf-52b31f526da8)

**Comment by ralfHielscher** [comment]
Hi,

1. Why does is not produce intended result?
2. Your code is in principle correct. However, I wonder why all Burgers vectors are considered equally. Wouldn't it make more sense to ignore very small Burgers vectors, as for those the axis is poorly defined anyway? Also wouldn't it make more sense, to weight the Burgers vectors according to the length when computing the distribution? For some reason those things are not mentioned in the description you gave.

A better code may be
```matlab
W = inv(ebsd.orientations) .* weightedBurgersVec(ebsd);

BurgerDistriFun = calcDensity(W,'weights',norm(W));
plot(BurgerDistriFun)
```

Ralf.
